src/components/Cards/CardLarge/CardLarge.js

```diff
@@ -0,0 +1,25 @@
+import React, { useRef } from 'react';
+
+import Image from '../../Image/Image';
+import Button from '../../Button/Button';
+import './CardLarge.scss';
+
+const CardLarge = ({ image, altTag, header, body, button, onClick }) => {
+  const ref = useRef();
+  return (
+    <div className='card-large'>
+      <Image className='card-large__image' img={image} alt={altTag} />
```

Comment:
you should use webp by default 
```suggestion
      <Image className='card-large__image' webp={image} alt={altTag} />
```